RepoData.json

```diff
@@ -0,0 +1,124 @@
+{
+    "fixedPackages": {
+        "coveralls.io": "1.4",
+        "MicroBuild.Core": "0.2.0",
+        "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00704-03",
+        "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040",
+        "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0040",
+        "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0040",
+        "Microsoft.DotNet.xunit.performance.runner.cli": "1.0.0-alpha-build0040",
+        "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0040",
+        "Microsoft.Net.Compilers": "2.0.0-beta3",        
+        "Microsoft.NETCore.Platforms": [ "1.0.1" ],
+        "Microsoft.NETCore.Targets": [ "1.0.0", "1.0.1" ],
+        "Microsoft.Portable.Targets": "0.1.1-dev",
+        "Microsoft.TargetingPack.Private.NETNative": "1.0.1-beta-24430-00",
+        "Microsoft.TargetingPack.Private.WinRT": "1.0.1",
+        "Microsoft.Win32.Primitives": [ "4.0.0" ],
+        "Microsoft.Win32.Registry": [ "4.0.0" ],
+        "Microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00704-03",
+        "Newtonsoft.Json": "8.0.4-beta1",
+        "OpenCover": "4.6.519",
+        "ReportGenerator": "2.4.3",
+        "System.AppContext": [ "4.0.0" ],
```

Comment:
Why are some of these arrays and some aren't? I think tooling should treat singleton arrays the same as a string, but I'm curious why in these cases.